repository/src/main/java/com/iluwatar/repository/Person.java

```diff
@@ -42,13 +43,19 @@
 
   private int age;
 
+  private static final int TOTALBRANCHES = 14;
+  public static boolean[] coveredBranches = new boolean[TOTALBRANCHES];
+  private boolean[] localCoveredBranches;
+
   public Person() {
+    localCoveredBranches = new boolean[TOTALBRANCHES];
   }
```

Comment:
You can instantiate this directly after private boolean[] localCoveredBranches and set every element to false in a BeforeEach or an AfterEach in the testclass